lib/index.js

```diff
@@ -75,7 +76,7 @@
         /* global document */
         var $ = modules.$;
         var Assertion = chai.Assertion;
-        Assertion.addMethod('accessible', function(config) {
+        Assertion.addMethod('accessible', function(config, ignoreGlobalRules) {
```

Comment:
Global Rules will be ignored even if nothing is passed (or if TRUE is passed) as there is an undefined check wrapping the portion of the code. To activate Global Rules - false will be have to passed. Let me know if this needs to be modified.